x/gov/endblocker.go

```diff
@@ -53,10 +53,19 @@ func EndBlocker(ctx sdk.Context, keeper Keeper) sdk.Tags {
 		passes, tallyResults := tally(ctx, keeper, activeProposal)
 
 		var tagValue string
+		var err error
 		if passes {
 			keeper.RefundDeposits(ctx, activeProposal.ProposalID)
 			activeProposal.Status = StatusPassed
-			tagValue = tags.ActionProposalPassed
+
+			handler := keeper.router.GetRoute(activeProposal.ProposalRoute())
+
+			err = handler(ctx, activeProposal.Content)
```

Comment:
handler is kind of a loaded name for a variable here, I'd recommend changing. Also I'm generally a tad confused as to what's happening here - there should probably be a comment. The fact as to whether a proposal passes or fails doesn't sound like a piece of information that should be passed back through an `err` variable.